electron/renderer/src/lib/localStorage.js

```diff
@@ -33,7 +32,7 @@ export const loadState = () => {
 export const saveState = (state) => {
```

Comment:
```js
export const saveState = state => {
```

Reply:
these things will be fixed automagically once the #1058 will be fixed and eventually Prettier (#1035)

Reply:
Well, yes, eventually...

Reply:
Maybe even tomorrow :) as @gregor has some ideas :) Robots will do their job, eventually..